OptimizelySDK.Tests/ProjectConfigTest.cs

```diff
@@ -51,7 +51,7 @@ public void Setup()
         public void TestInit()
         {
             // Check Version
-            Assert.AreEqual(2, Config.Version);
```

Comment:
Until we GA feature flags, we need to support BOTH datafile v2 and datafile v4.

Reply:
Nah because we can keep this in the master branch and deploy from the 1.2.x branch of we need to do so before FF GA.